build.sh

```diff
@@ -17,6 +17,7 @@ BUILD_DIR=${BUILD_DIR:-${UFS_MODEL_DIR}/build}
 mkdir -p ${BUILD_DIR}
 
 [[ -n "${CCPP_SUITES:-""}" ]] && CMAKE_FLAGS+=" -DCCPP_SUITES=${CCPP_SUITES}"
+[[ -n "${MAPL_ROOT:-""}" ]] && CMAKE_FLAGS+=" -DCMAKE_MODULE_PATH=${MAPL_ROOT}/share/MAPL/cmake"
```

Comment:
This location (`${MAPL_ROOT}/share/MAPL/cmake`) should be added to `CMAKE_MODULE_PATH` somewhere in some CMakeLists.txt in gocart code so that people can build the code without running this build script i.e. by just running cmake (and of course loading required modules where necessary).

Reply:
MAPL is installing the `mapl-config.cmake` in a non-standard path at:
```
<prefix>/share/MAPL/cmake
```

The standard paths that CMake will look for given `PACKAGE_ROOT` as `prefix` are:
```
<prefix>/(lib/<arch>|lib|share)/cmake/<name>*/          (U)
<prefix>/(lib/<arch>|lib|share)/<name>*/                (U)
<prefix>/(lib/<arch>|lib|share)/<name>*/(cmake|CMake)/  (U)
```

See:
https://cmake.org/cmake/help/v3.0/command/find_package.html